### PR TITLE
Add support for Django 4.0

### LINKED ISSUE #14: Support for Django 4
Currently, this package supports Django until the versions 3.2. Is it possible to add a support for [Django 4.0](https://docs.djangoproject.com/en/4.0/releases/4.0/)?

### PR DESCRIPTION
**Pull request by benoit9126**
Try to close #14 

@llybin  I would also propose to update the `django-ipware` dependency as in version 4.0.1, the creator of this library added tests on Django 4.0. See [changelog.md](https://github.com/un33k/django-ipware/blob/master/CHANGELOG.md). If you agree, I can add another commit in this PR.

In a third commit, it is also possible to add the version of Python 3.10 in the tests, `setup.py` file and `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 ## Requirements
 
 *   Python: 3.6, 3.7, 3.8, 3.9
```
Comment (llybin) [inline README.md:15]: 3.10 +

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.6", "3.7", "3.8", "3.9"]
```
Comment (llybin) [inline .github/workflows/tests.yml:12]: https://github.com/encode/django-rest-framework/blob/master/tox.ini#L6 looks like need to exclude python 3.6 3.7 for django4
yep, also can add 3.10 and exclude that for "2.2", "3.0", "3.1"
also I see DRF stopped testing for 3.0 version, can remove everywhere 3.0 too

**Review by llybin — COMMENTED** [review]
And upgrade version to 2.0.6 please https://github.com/llybin/drf-recaptcha/pull/15/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R10

In next PR 2.0.7 yes can upgrade django-ipware. thanks!

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -26,6 +22,15 @@ jobs:
             drf-version: "3.9"
           - django-version: "3.2"
             drf-version: "3.10"
+          - django-version: "4.0"
+            drf-version: "3.9"
+          - django-version: "4.0"
+            drf-version: "3.10"
+          # python 3.10 is incompatible with django "2.2", "3.0", "3.1"
```
Comment (llybin) [inline .github/workflows/tests.yml:29]: remove 3.0

File: README.md [hunk 4]
```diff
@@ -12,8 +12,8 @@
 
 ## Requirements
 
-*   Python: 3.6, 3.7, 3.8, 3.9
-*   Django: 2.2, 3.0, 3.1, 3.2
+*   Python: 3.6, 3.7, 3.8, 3.9, 3.10
+*   Django: 2.2, 3.0, 3.1, 3.2, 4.0
```
Comment (llybin) [inline README.md:16]: remove 3.0

File: setup.py [hunk 5]
```diff
@@ -45,6 +45,7 @@ def read(f):
         "Framework :: Django :: 3.0",
```
Comment (llybin) [inline setup.py:45]: remove 3.0 and great

**Comment by benoit9126** [comment]
@llybin It should be good

**Review by llybin — APPROVED** [review]

**Pull request merged**